### PR TITLE
Swap annotation and object properties panel

### PR DESCRIPTION
**Pull request by trungleduc**
This PR swaps the position of the annotation panel and the object properties panel.

cad222

### CODE REVIEW

**Comment by trungleduc** [comment]
Bot please update snapshots!

**Comment by trungleduc** [comment]
Bot please update snapshots!

File: python/jupytercad_lab/src/index.ts [hunk 1]
```diff
@@ -89,11 +89,11 @@ const controlPanel: JupyterFrontEndPlugin<void> = {
     formSchemaRegistry: IJCadFormSchemaRegistry
   ) => {
     const controlModel = new ControlPanelModel({ tracker });
-
+    console.log('annotationModel', annotationModel);
```
Comment (arjxn-py) [inline python/jupytercad_lab/src/index.ts:92]:
```suggestion
```

**Comment by SylvainCorlay** [comment]
This looks good to me.

**Review by arjxn-py — APPROVED** [review]
Thanks @trungleduc, looks awesome works awesome 🚀

**Pull request merged**